### PR TITLE
UPSTREAM: docker/distribution: 2140: Add 'ca-central-1' region for registry S3 storage driver

### PR DESCRIPTION
**Pull request by mfojtik**
See: https://github.com/docker/distribution/pull/2140

### CODE REVIEW

**Comment by mfojtik** [comment]
(need to verify this actually works for the new regions)

**Comment by mfojtik** [comment]
(this doesn't work without bumping the aws-sdk-go dependency or hardcoding the fields in the endpoints there, @miminar is currently investigating this)

**Comment by miminar** [comment]
I've just verified this works against `ca-central-1` (aka Montreal) region. I can push and pull there without a problem.

File: vendor/github.com/docker/distribution/registry/storage/driver/s3-aws/s3.go [hunk 1]
```diff
@@ -75,14 +75,17 @@ func init() {
 		"us-west-1",
```
Comment (miminar) [inline vendor/github.com/docker/distribution/registry/storage/driver/s3-aws/s3.go]: Add `us-east-2` as well please. I've tested it - works like a charm.

Comment (mfojtik) [inline vendor/github.com/docker/distribution/registry/storage/driver/s3-aws/s3.go]: will do

**Comment by mfojtik** [comment]
[merge]

**Comment by mfojtik** [comment]
[merge]

**Pull request merged**